src/main/java/net/pms/dlna/DLNAResource.java

```diff
@@ -1009,7 +1009,10 @@ public Player resolvePlayer(RendererConfiguration renderer) {
 			// 2) transcoding is preferred and not prevented by configuration
 			if (forceTranscode || (preferTranscode && !isSkipTranscode())) {
 				if (parserV2) {
-					LOGGER.trace("Final verdict: \"{}\" will be transcoded with player \"{}\" with mime type \"{}\"", getName(), resolvedPlayer.toString(), renderer != null ? renderer.getMimeType(mimeType(resolvedPlayer), media) : media.getMimeType());
+					LOGGER.trace("Final verdict: \"{}\" will be transcoded with player \"{}\" with mime type \"{}\"",
+						getName(),
+						resolvedPlayer.toString(),
```

Comment:
```toString()``` isn't needed when calling the logger, it will be invoked by the logger as needed. The advantage of letting the logger do it is that ```toString()``` won't be called if the log level isn't "trace".